tests/integration/issues/github_969/test_messages_different_types.py

```diff
@@ -0,0 +1,291 @@
+import pytest
+from jina.flow import Flow
+import numpy as np
+from jina.proto import jina_pb2
+from jina.drivers.helper import array2pb, pb2array
+
+random_np_array = np.random.randint(10, size=(50, 10))
+buffer = 'text_buffer'.encode()
+text = 'text_content'
+
+
+@pytest.mark.skip(reason='with optimization, message cannot handle different chunk types in the same message')
```

Comment:
If you expect an error to happen, I'd rather use `pytest.raises` instead of skipping the test. This guarantees, if we fix the problem, we don't forget to activate the test.

If it is not fixable: remove the test.

Reply:
I think having a raised error and test against it can lead to the impression that the test must fail, this part is a preparation for the future

Reply:
:+1: